BLE_PeriodicAdvertising/source/main.cpp

```diff
@@ -408,32 +387,37 @@ class PeriodicDemo : private mbed::NonCopyable<PeriodicDemo>, public ble::Gap::E
             const disconnect_call_t disconnect_call = &Gap::disconnect;
 
             if (_is_scanner) {
+                printf("I will synchronise with periodic advertising\r\n");
                 _event_queue.call_in(
-                    2000,
+                    1000ms,
```

Comment:
A lambda is much better here and it removes the need for typedef above. : 

```c++
_event_queue.call_in(1000ms, [this, handle = event.getConnectionHandle()] {
    _ble.gap.disconnect(handle, ble::local_disconnection_reason_t::USER_TERMINATION);
});
```

Reply:
thanks